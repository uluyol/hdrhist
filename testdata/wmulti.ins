0
1
555
21
8
132123
11
---
1
2
789
9
1
---
2
3
---
3
4
0
---
4
5
1
---
