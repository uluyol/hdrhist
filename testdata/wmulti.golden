0.000,0.001,0.132,HISTFAAAACp42pNpmSzMwMDAwwABzFCaEcL7/9/+A0SAn4mZSZBpJQfT9nImAI3SBpM=
0.001,0.001,0.001,HISTFAAAACJ42pNpmSzMwMDAzgABzFCaEUoz2X+AsXiZpvIwAQBO1QM3
0.002,0.001,0.000,HISTFAAAABx42pNpmSzMwMDAyAABzFAaxmey/wBlAQA8yQJ9
0.003,0.001,0.000,HISTFAAAABx42pNpmSzMwMDAyAABzFAaxmey/wBlAAA8ywJ/
0.004,0.001,0.000,HISTFAAAABx42pNpmSzMwMDAxAABzFCaEUoz2X+AsQA/awKA
